src/components/errors/friendly-error.tsx

```
@@ -0,0 +1,404 @@
+    </div>
+  );
+}
+
+const NotFoundIllustration = () => (
+  <svg
+    width="248"
+    height="180"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    className="opacity-50 ml-6"
+  >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)